packages/appium/docs/en/cli/args.md

```diff
@@ -2,4 +2,54 @@
 title: Server CLI Args
 ---
 
-TODO
+To start the Appium server, you may either run `appium` or `appium server`. The `server` subcommand
+is considered to be the default, so if you omit it, Appium will interpret this as your request to
+run the Appium server. (You could run different subcommands, like `appium driver` or `appium
+plugin`: see the [Extensions CLI](./extensions.md) for more info).
+
+The invocation of `appium` (or `appium server`) takes a number of arguments, which are detailed
+below.
+
+!!! note
```

Comment:
how about https://twitter.com/diegohaz/status/1527642881384759297?lang=en ?

Reply:
that's awesome! this text won't primarily be consumed on github, though, so for now we still need to use the mkdocs-material supported syntax until they support the github syntax